storm-core/src/clj/backtype/storm/daemon/executor.clj

```diff
@@ -410,7 +410,11 @@
     (disruptor/clojure-handler
       (fn [tuple-batch sequence-id end-of-batch?]
         (fast-list-iter [[task-id msg] tuple-batch]
-          (let [^TupleImpl tuple (if (instance? Tuple msg) msg (.deserialize deserializer msg))]
+          (let [[deserialize-time ^TupleImpl tuple] (if (instance? Tuple msg) [0.0 msg] (with-time (.deserialize deserializer msg)))]
```

Comment:
In all other cases the timings were sampled (we didn't measure everything).  Not totally sure if we can do the same thing here or not.